x-pack/plugins/observability_solution/apm/public/components/routing/home/index.tsx

```diff
@@ -50,11 +49,13 @@ function serviceGroupPage<TPath extends string>({
   return {
     [path]: {
       element: (
-        <Breadcrumb title={title} href={path}>
```

Comment:
>[!NOTE]
<Breadcrumb /> was removed to avoid conflicts with `useBreadcrumb` in `ServiceGroupTemplate`

Reply:
totally off topic: never knew about these alert indicators. I'll immediately start (ab)-using these! :)

Other type of alerts per my recent findings:

> [!NOTE]  
> Highlights information that users should take into account, even when skimming.

> [!TIP]
> Optional information to help a user be more successful.

> [!IMPORTANT]  
> Crucial information necessary for users to succeed.

> [!WARNING]  
> Critical content demanding immediate user attention due to potential risks.

> [!CAUTION]
> Negative potential consequences of an action.

Reply:
I have to say that I found that out a couple of days ago in one of @tonyghiani's PRs. 😄